### ISSUE TITLE
"SCE Object yet to create" even when object supplied to iSEE()

### ISSUE THREAD

**Issue by kevinrue** [csoneson/iSEE#32]

**Comment by LTLA** [comment]
I believe this functionality should be taken out altogether. I can't imagine a scenario where people would want to upload SCE objects to a webpage; they would be better off instantiating a new Shiny instance from the R terminal via a new `iSEE` call. If they had enough know-how to get a SCE object in the first place, this shouldn't be a problem.

**Comment by kevinrue** [comment]
That's true.
The `CellView` (https://mbolisetty.shinyapps.io/CellView/) tool offers such an 'upload RDS file' feature, which allows a 'blank' app to be hosted online. However, I think this overcomplicating a simple situation when considering a single project at a time.

From discussion in the lab, web-hosting `iSEE` to share data with collaborators who don't necessarily have R is an attractive and even desirable feature, I think it can be done by deploying a distinct app for each project, and launching e.g. `iSEE(readRDS(...))` with a different RDS file in each app for instance.

Bottom line is I agree there is little/no benefit in launching an empty app.

**Comment by LTLA** [comment]
I think I erased the box completely in a UI update, so I'm closing this.
